src/platforms/node/common/integrations/default-integrations.mdx

```diff
@@ -5,12 +5,18 @@ redirect_from:
   - /platforms/node/default-integrations/
 ---
 
-System integrations are integrations enabled by default that combine into the
+System integrations are integrations enabled by default that hook into the
 standard library or the interpreter itself. They're documented so you can see
-what they do and that they can be disabled if they cause issues. To disable
-system integrations set `defaultIntegrations: false` when calling `init()`.
-To override their settings, provide a new instance with your config
-to `integrations` option. For example: to change fatal error handler
+what they do and know how to disable them if they cause issues. 
+
+To disable _all_ default integrations, set `defaultIntegrations: false` when calling `init()`.
+
+To disable just one integration, provide a filtered list to the `integrations` option when calling `init()`.
+For example, to disable the `Console` integration:
+`integrations: defaults => defaults.filter(integration => integration.name !== 'Console')`.
+
+To override an integration's settings, provide a new instance to the `integrations` option when calling `init()`.
+For example, to change the fatal error handler:
```

Comment:
There are so many ways to use the same `integrations` option -- `bool`, `func`, `array`.

We document using each way for one specific need. What's missing here is how to combine the needs, how to both provide specific integration settings AND remove/add integrations.

I'm guessing the answer is to use the `function` form, which seems to be the most powerful.
In fact, it could be used for all three examples:

```
// remove all integrations:
integrations: () => null; // or if we don't support this, then empty array

// remove one or more filtering by name:
integrations: defaults => defaults.filter(...)

// replace an integration with a custom version of the same integration:
integrations: defaults => defaults.map(integration =>
  integration.name === 'OnUncaughtException' ?
    new Sentry.Integrations.OnUncaughtException({ onFatalError: () => { /** your implementation */ } }) :
    integration)

// filter and update:
integrations: defaults => ...
```

Reply:
> // remove all integrations:

It should be just `defaultIntegrations: false` then, which is already described slightly before I guess.

> // replace an integration with a custom version of the same integration:

We always tell people to use `integrations: [new VersionWithOptions({})]` as it's more readable for this basic usecase

Reply:
@rhcarvalho I think that you're right, and perhaps we should revisit this again in the future (and unify our explanation here with the one on the main JS SDK page), but for the moment I'm going to merge this and we can come back to it.